publish.msbuild

```diff
@@ -3,8 +3,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <Import Project="Directory.Build.targets" />
 
-  <Import Project="$(NuGetPackageRoot)/$(FeedTasksPackage)/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
```

Comment:
Looks like this file is using its own targets to wrap the `PushToBlobFeed` task.  Any reason you can't just use the targets from the package?

Reply:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.Feed/build/Microsoft.DotNet.Build.Tasks.Feed.targets

Reply:
This file should be entirely redundant once the pipeline is using Arcade's publish task, I'm planning on deleting it once I've validated that everything works.

Reply:
That is, we're currently not using it anywhere, I'm just waiting to delete it until I've convinced myself we definitely won't need it anymore.